CHANGELOG.md

```diff
@@ -2,15 +2,28 @@
 
 ## **next**
 
-### Enhancements
+### Enhancements)
+
+### Bug fixes
+
+### Released packages
```

Comment:
Is it really necessary to add this?

Reply:
Just the `Released packages` or all the `next` section?

Reply:
All the lines I commented or the whole `next` section, as you wish.